lighthouse-core/aggregator/aggregate.js

```diff
@@ -75,16 +75,15 @@ class Aggregate {
     if (typeof expected === 'undefined' ||
         typeof expected.rawValue === 'undefined' ||
         typeof expected.weight === 'undefined') {
-      return weight;
+      throw new Error('Expected is undefined or does not contain rawValue and weight properties');
     }
 
     if (typeof result === 'undefined' ||
-        typeof result.rawValue === 'undefined' ||
         typeof result.score === 'undefined') {
-      return weight;
+      throw new Error('Result is undefined or does not contain score property');
```

Comment:
same as above. Should we surface `debugString` at this point if it exists? That way it's at least clear the issue is in the audit or gatherer